crc/xxhash.h

```diff
@@ -107,9 +107,9 @@ XXH32() :
 // Advanced Hash Functions
 //****************************
```

Comment:
`crc/xxhash.c:263:15: error: conflicting types for 'XXH32_resetState'
 XXH_errorcode XXH32_resetState(void* state_in, uint32_t seed)
               ^~~~~~~~~~~~~~~~
In file included from crc/xxhash.c:65:0:
crc/xxhash.h:138:15: note: previous declaration of 'XXH32_resetState' was here
 XXH_errorcode XXH32_resetState(void* state, unsigned int seed);
               ^~~~~~~~~~~~~~~~
crc/xxhash.c:277:7: error: conflicting types for 'XXH32_init'
 void* XXH32_init (uint32_t seed)
       ^~~~~~~~~~
In file included from crc/xxhash.c:65:0:
crc/xxhash.h:110:15: note: previous declaration of 'XXH32_init' was here
 void*         XXH32_init   (unsigned int seed);` 
No, the issue is still there.